lib/prism/ripper_compat.rb

```diff
@@ -203,6 +228,21 @@ def visit_statements_node(node)
       end
     end
 
+    private
+
+    # Ripper generates an interesting format of argument list.
+    # It seems to be very location-specific. We should get rid of
+    # this method and make it clearer how it's done in each place.
+    def args_node_to_arguments(args_node)
```

Comment:
Just for my own paranoia could you move this down to the rest of the `private` methods? I know it's pedantic, but I'm always fearful of accidentally exposing methods we don't mean to.

Reply:
I've actually removed it entirely in my next change.

Reply:
Okay, moved it in this PR. But either way, disappearing soon.